src/main/java/easytests/api/v1/controllers/IssuesController.java

```diff
@@ -65,6 +65,22 @@
     /**
      * show(issueId)
      */
+
+    @GetMapping("/{issueId}")
+    public Object show(@PathVariable Integer issueId) throws NotFoundException, ForbiddenException {
+        final IssueModelInterface issueModel = this.getIssueModel(issueId);
```

Comment:
where is forbidden check?
provide test also

Reply:
@malinink  Waiting for @miron97 's pr to be accepted to ese  ACL for issues. Or should I implement it by myself?

Reply:
Ok, that Pr is almost done. I will soon merge it.